tests/test/org/commcare/util/test/UserSqlSandboxTest.java

```diff
@@ -0,0 +1,43 @@
+package org.commcare.util.test;
+
+import org.commcare.api.persistence.UserSqlSandbox;
+import org.commcare.api.persistence.SqlSandboxUtils;
+import org.commcare.cases.ledger.Ledger;
+import org.commcare.cases.model.Case;
+import org.commcare.core.parse.ParseUtils;
+import org.javarosa.core.model.User;
+import org.commcare.test.utils.SqlTestUtils;
+import org.javarosa.core.model.instance.FormInstance;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Vector;
+
+/**
+ * Tests for the SqlSandbox API. Just initializes and makes sure we can access at the moment.
+ *
+ * @author wspride
+ */
+public class UserSqlSandboxTest {
+
+    private UserSqlSandbox sandbox;
+    private Vector<String> owners;
+    String username = "sandbox-test-user";
+
+    @Before
+    public void setUp() throws Exception {
+        SqlTestUtils.deleteDatabase(username);
+        sandbox = SqlSandboxUtils.getStaticStorage(username);
+        ParseUtils.parseIntoSandbox(this.getClass().getClassLoader().getResourceAsStream("ipm_restore.xml"), sandbox);
```

Comment:
This statement is printing a lot of stack traces in the test suite. I would look into those, as they probably indicate a bug.